### DISCUSSION TITLE
Relaxing JSON Parsing

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/runtime#93976] [Q&A]
I am doing some work with the Sharepoint JSON APIS and I've found that sometimes the APIs do not return properly encoded JSON.  Specifically, the 0x01 (SOH) character is listed directly in some string content and not encoded/escaped.

Is there a way for me to relax STJ's parsing of this? Alternatively, is there a way to get a bug report in front of someone on the sharepoint  online team so this can be fixed?

**Comment by Lightczx** [comment]
You can implement your own `JavaScriptEncoder` and replace the one in `JsonSerializerOptions`
